stories/core/Fieldset.stories.tsx

```diff
@@ -0,0 +1,92 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React, { useState } from 'react';
+import {
+  Fieldset,
+  FieldsetProps,
+  InputGroup,
+  LabeledSelect,
+  Radio,
+  ToggleSwitch,
+} from '../../src/core';
+
+export default {
+  title: 'Input/Fieldset',
+  component: Fieldset,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
```

Comment:
I think these two lines can be removed.

Reply:
Really? We are disabling these fields for every story.

Reply:
They don't show up, so they don't need to be disabled 🤔